config/routes.rb

```diff
@@ -62,6 +62,7 @@
 
   resources :students, only: [:show] do
     resources :event_notes, only: [:create, :update]
+    resources :student_photos, only: [:show]
```

Comment:
This looks like it requires the UI to know the `student_photo_id`.  Using something like `/students/:id/photo` might be easier for us to work with when making UI and product developers, but that's probably also a question for another PR :)

Reply:
Ooooh no you're totally right. Thought about this as soon as I woke up this morning! No need to pull the student's latest photo's ID out of the database, we already know which photo we want.